_targets/build.project.armv7a7-imx6ull

```diff
@@ -113,26 +127,47 @@ b_image_target() {
 	b_log "Creating image from $PREFIX_ROOTFS"
 
 	b_mkscript_user "${UPLOAD_SCRIPT[@]}"
-	b_mkscript_preinit
-
-	# Build production image
-	b_log "RAM Image"
-	b_kernel_image "$PREFIX_BOOT/phoenix.img"
-	b_syspage_gen "$PREFIX_BOOT/phoenix.img" "$PLO_SCRIPT_DIR/$NAME_USER_SCRIPT"
 
-	b_log "FLASH Image"
-	b_mkscript_user "${PRIMARY_SCRIPT[@]}"
-	b_kernel_image "$PREFIX_BOOT/phoenix-kernel.img"
-	b_syspage_gen "$PREFIX_BOOT/phoenix-kernel.img" "$PLO_SCRIPT_DIR/$NAME_USER_SCRIPT"
+	if [ "$KERNEL_PLO_BOOT" = "y" ]; then
+		b_log "Image will boot from PLO."
```

Comment:
IMHO the plo should be part of the disk image (I'm assuming currently it's not?) Probably kernel offset in the disk image would need to be changed, not sure if we want to introduce `ptable` at this stage or not

Reply:
why would like to have PLO on targets not using it?

Reply:
I agree that writing full disk images is wasteful. However I am not sure how big of an issue it is as the gaps between parts should be as small as possible both in full images and in splitted smaller images, so  amount of zeroed block shouldn't be higher that few. @agkaminski what's your take on that matter?

Reply:
After some more thinking:
 - the disk image can't realistically be used for NAND use case (we can't write `0xff` as a no-op, we need to check/skip badblocks, etc.) so maybe let's not focus on that (should we add some comment?)
 - for NOR `disk` image for sure is useful, but all free space should be marked as `0xff` - the paddings might be considerable (if ptable would be added to the disk image - it usually resides at the end of the flash)

Sample code for `0xff` padding generation (can be done in other ways):
```bash
pad_ff()
{
	dd if=/dev/zero ibs=1 count="$2" 2>/dev/null | tr "\000" "\377" > padding.img
	dd if="$1" of=padding.img conv=notrunc 2>/dev/null
	mv padding.img "$1"
}
```